test/rustler_precompiled_test.exs

```diff
@@ -910,7 +1015,7 @@ defmodule RustlerPrecompiledTest do
                  "libexample-v0.2.0-nif-2.17-x86_64-unknown-linux-gnu.so.tar.gz",
                  "libexample-v0.2.0-nif-2.17-x86_64-unknown-linux-musl.so.tar.gz"
                ]
-               |> Enum.map(fn file_name -> "#{base_url}/#{file_name}" end)
+               |> Enum.map(fn file_name -> {file_name, {"#{base_url}/#{file_name}", []}} end)
```

Comment:
This is the public API change as mentioned [above](https://github.com/philss/rustler_precompiled/pull/77/files#r1652647852)

Reply:
Yeah, as I mentioned, I think we could keep the same API for the current function, and add a new one with this new return. Then we deprecate the old one (the tests can follow the new one only).